### PR TITLE
Update @fedify/cli for Optique 1.0.0

### PR DESCRIPTION
**Pull request by dahlia**
## Summary

- Bump the shared Optique dependencies to 1.0.0 in *deno.json*, *pnpm-workspace.yaml*, and the lockfiles.
- Switch *packages/cli/src/mod.ts* from `runWithConfig()` to `run()`, pass `contexts` and `contextOptions`, and update the help/completion wiring for Optique 1.0.
- Adjust CLI parser tests in *packages/cli/src/lookup.test.ts*, *packages/cli/src/tunnel.test.ts*, and *packages/cli/src/webfinger/mod.test.ts*.
- Fix a runtime issue in *packages/init/src/action/configs.ts* found during `fedify` startup smoke tests.

## Why

Optique 1.0.0 removes `runWithConfig()` and replaces it with the finalized runner and context API. This keeps the Fedify CLI compatible with the 1.0 release without changing config loading or shell completion behavior.

Release notes: https://github.com/dahlia/optique/discussions/796

## Testing

- `deno task -f @fedify/cli test`
- `node --test --experimental-transform-types 'src/**/*.test.ts' '!src/init/test/**'` in *packages/cli*
- `pnpm test` in *packages/init*
- `node --disable-warning=ExperimentalWarning dist/mod.js --help` in *packages/cli*
- `node --disable-warning=ExperimentalWarning dist/mod.js completions bash` in *packages/cli*
- `node dist/test/mod.js --help` in *packages/init*
- `node dist/mod.js --help` in *packages/create*

### CODE REVIEW

File: packages/cli/src/tunnel.test.ts [hunk 1]
```diff
@@ -1,15 +1,15 @@
 import type { Tunnel, TunnelOptions } from "@hongminhee/localtunnel";
-import { run } from "@optique/run";
+import { runSync } from "@optique/run";
 import { deepEqual, rejects } from "node:assert/strict";
 import test from "node:test";
```
[the remark this answers is not in the document]
Comment (dahlia) [reply packages/cli/src/tunnel.test.ts:4]: No, it's perfectly fine with the `node:test` module. We should update the guideline instead.

[the remark this answers is not in the document]
Comment (dahlia) [reply packages/cli/src/tunnel.test.ts:4]: Kept node:test unchanged. The CLI package tests are intentionally Node-specific here, and dahlia already confirmed that in this thread.

File: packages/cli/src/lookup.test.ts [hunk 2]
```diff
@@ -31,6 +30,21 @@ import {
...
+): Promise<Result<TValue>> {
+  const annotations = await (configContext.getAnnotations as (
+    request?: unknown,
+    options?: unknown,
+  ) => PromiseLike<Annotations> | Annotations)(
+    { phase: "phase2", parsed: {} },
+    { load: () => ({ config, meta: undefined }) },
+  );
+  return parse(parser, args, { annotations });
+}
```
[the remark this answers is not in the document]
Comment (dahlia) [reply packages/cli/src/lookup.test.ts:52]: Fixed in 689d861b. I split the CLI runner setup into a shared helper and added regression tests that exercise the real run() + contextOptions.load path for both --config and --ignore-config.

File: packages/cli/src/lookup.test.ts [hunk 3]
```diff
@@ -31,6 +30,21 @@ import {
...
 
+async function parseWithConfig<TValue, TState>(
+  parser: Parser<"sync", TValue, TState>,
+  args: readonly string[],
+  config: Record<string, unknown> = {},
+): Promise<Result<TValue>> {
+  const annotations = await (configContext.getAnnotations as (
+    request?: unknown,
+    options?: unknown,
+  ) => PromiseLike<Annotations> | Annotations)(
```
[the remark this answers is not in the document]
Comment (dahlia) [reply packages/cli/src/lookup.test.ts:47]: I kept the parser-only helper for the narrow annotation-level assertions, but I documented why it has to emulate phase 2 and added runner-level coverage in 689d861b so the real Optique load path is exercised as well.

File: packages/init/src/action/configs.ts [hunk 4]
```diff
@@ -59,26 +57,30 @@ export const loadDenoConfig = (data: InitCommandData) => ({
...
+  const version = pipe(
     getDenoVersionFromRuntime(),
     when(isNull, getDenoVersionFromCommand),
     when(isString, parseVersion),
-    cases(isArray, isLaterOrEqualThan(TEMPORAL_STABLE_FROM), always(true)),
   );
+  return isArray(version)
+    ? isLaterOrEqualThan(TEMPORAL_STABLE_FROM)(version)
+    : true;
+};
```
[the remark this answers is not in the document]
Comment (dahlia) [reply packages/init/src/action/configs.ts:83]: Fixed in 689d861b. The issue was both the comparison direction and version parsing: parseVersion was including the full regex match, so 2.7.0 was being interpreted incorrectly. I corrected both and added regression coverage in packages/init/src/action/configs.test.ts.

**Comment by dahlia** [comment]
For the outside-diff suggestion in CodeRabbit's review summary about switching packages/cli/src/lookup.test.ts to @fedify/fixture, I kept node:test there as well. The CLI package tests are intentionally Node-specific, the same as in the tunnel test thread.

File: packages/init/src/action/configs.test.ts [hunk 5]
```diff
@@ -0,0 +1,84 @@
+import assert from "node:assert/strict";
+import test from "node:test";
```
[the remark this answers is not in the document]
Comment (dahlia) [reply packages/init/src/action/configs.test.ts:2]: No, it's totally fine. See also <https://github.com/fedify-dev/fedify/pull/677#discussion_r3083460277>.

File: packages/cli/src/runner.ts [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+        group: "Meta commands",
+      },
+    },
+    colors: process.stdout.isTTY &&
+      (process.env.NO_COLOR == null || process.env.NO_COLOR === ""),
+    maxWidth: process.stdout.columns,
+    showDefault: true,
+    showChoices: true,
+  });
+}
```
[the remark this answers is not in the document]
Comment (dahlia) [reply packages/cli/src/runner.ts:154]: Addressed in 4b28b9af. I added a short JSDoc block to runCli() since it is now the shared entrypoint for the Optique 1.0 runner setup.

File: packages/init/src/action/configs.test.ts (hunk 5 above)
[the remark this answers is not in the document]
Comment (dahlia) [reply packages/init/src/action/configs.test.ts:2]: Left this unchanged. As dahlia noted above, packages/init/src tests intentionally use node:test here, so I did not switch this file to @fedify/fixture.

**Comment by dahlia** [comment]
For CodeRabbit's outside-diff note about missing await in packages/cli/src/mod.ts, I fixed that in 4b28b9af by awaiting both runInbox() and runNodeInfo().

File: packages/init/src/action/configs.test.ts [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+    mq: {
+      label: "Deno KV",
+      packageManagers: ["deno"],
+      imports: {},
+      object: "mq",
+      denoUnstable: [],
+    },
+    env: {},
+  } as unknown as InitCommandData;
+}
```
[the remark this answers is not in the document]
Comment (dahlia) [reply packages/init/src/action/configs.test.ts:42]: Fixed in fd47b772. The test now builds a real InitCommandData object with satisfies InitCommandData, adds the missing command field, removes properties that are not part of the type, and uses message`done` instead of the double cast.

File: packages/cli/src/inbox/command.ts [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+        },
+      ),
+    }),
+    group("Tunnel options", createTunnelOption("inbox")),
+  ),
+  {
+    brief: message`Run an ephemeral ActivityPub inbox server.`,
+    description:
+      message`Spins up an ephemeral server that serves the ActivityPub inbox with an one-time actor, through a short-lived public DNS with HTTPS. You can monitor the incoming activities in real-time.`,
+  },
```
[the remark this answers is not in the document]
Comment (dahlia) [reply packages/cli/src/inbox/command.ts:96]: Fixed in fd47b772. I corrected the user-facing inbox description to say "a one-time actor" and kept the rest of the wording unchanged.

File: packages/init/src/action/configs.test.ts [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+  const originalDeno = (globalThis as Record<string, unknown>).Deno;
+  setDenoVersion({ deno: "2.6.9", v8: "0.0.0", typescript: "0.0.0" });
+
+  try {
+    const config = loadDenoConfig(createInitData()).data;
+    assert.deepStrictEqual(config.unstable, ["temporal"]);
+  } finally {
+    restoreDeno(originalDeno);
+  }
+});
```
[the remark this answers is not in the document]
Comment (dahlia) [reply packages/init/src/action/configs.test.ts]: Fixed in 86a1f54b. I kept the test itself unchanged and narrowed the title to the exact 2.7.0 boundary case it asserts, so the wording now matches the covered matrix.

File: packages/cli/src/inbox/command.ts [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+        option("--actor-summary", string({ metavar: "SUMMARY" }), {
+          description: message`Customize the actor description.`,
+        }),
+        {
+          context: configContext,
+          key: (config) =>
+            config.inbox?.actorSummary ??
+              "An ephemeral ActivityPub inbox for testing purposes.",
+          default: "An ephemeral ActivityPub inbox for testing purposes.",
+        },
```
[the remark this answers is not in the document]
Comment (dahlia) [reply packages/cli/src/inbox/command.ts]: Fixed in 86a1f54b. I extracted the repeated inbox actor fallback strings into shared constants so the bindConfig fallback and default values stay aligned.
